client/src/components/ScoreForm.js

```diff
@@ -80,11 +136,22 @@ const TableBonuses = ({ bonuses, handleCheckedBox }) => (
 	</Table>
 )
 
-const CurrIntegerExercise = ({ value, units }) => (<div>{value} {units}</div>)
+const Bonus = field => (
+	<Checkbox
+		{...field.input}
+		value={field.input.value ? 'on' : 'off'}
+		onChange={(e, { checked }) => field.input.onChange(checked)}
+		label={field.label} 		
+	/>
+)
+
+const CurrIntegerExercise = ({ currValue, units }) => <div>{currValue} {units}</div>
```

Comment:
We probably don't need a standalone component when it's so simple and being used in one area. We can refactor and place it directly where it's used in line 56. 

Same thing with ```const NewIntegerExercise```. We can refactor and place directly where its used in line 67.

Reply:
I removed the CurrentIntegerExercise, NewIntegerExercise, and CurrentIntervalExercise components and placed them directly where they were being used.

I kept NewIntervalExercise component as that was being used twice.